src/Components/ComponentLockModel.cs

```diff
@@ -14,6 +14,7 @@ namespace Vasont.Inspire.Models.Components
     /// <summary>
     /// Contains an enumerated list of component lock types.
     /// </summary>
+    [Obsolete("Use local enums", false)]
```

Comment:
As I understand it, specifying "false" for the error parameter indicates that this object is now deprecated and to be obsolete by the next major release. Should we update the message to specify that it is being deprecated, which local object to use, and when it will be obsoleted?